cli/src/commands/backend/mod.rs

```diff
@@ -1,15 +1,17 @@
-use std::{collections::HashMap, path::PathBuf};
+use std::{collections::HashMap, io::Write, path::PathBuf};
 
 use clap::Parser;
-use cli_core::rivet_api::apis;
-use console::Term;
+use cli_core::rivet_api::{apis, models};
 use global_error::prelude::*;
 use serde::Deserialize;
+use tempfile::NamedTempFile;
 use tokio::fs;
 use tokio::process::Command;
 
 use crate::util::{global_config, paths, term};
 
+const DEFAULT_OPENGB_DOCKER_TAG: &'static str = "ghcr.io/rivet-gg/opengb/v0.1.2";
```

Comment:
How did you manage to get ghcr working publicly? Iirc that was the issue with using it for k8s in bolt

Reply:
oh i should double check this actually works e2e without auth, thx.